### DISCUSSION TITLE
Permission denied while running under docker

### DISCUSSION THREAD

**Discussion by MikeyMan83** [pymedusa/Medusa#10188] [Q&A]
Don't really know if this is a DSM7, a docker or a Medusa question, so i'm searching my ass off ;)

Situation: Installed Medusa on my new DS1520+, had to use docker because i was forced to update to DSM7.
Now, most is running quite well after a very (VERY!) steep learning curve. 

Created a Docker user (1040:100), and gave that user access to all folders i need. These were previously created in DSM6 (Or even 5?).
Now Medusa is able to download and move shows as before. 

However; when i try to remove a show, it gives the following error:
SHOWQUEUE-REMOVE-SHOW :: 247897: Unable to delete /Series/Homeland. Error: Error: 13: Permission denied

No clue what other permissions i am lacking, since writing seems to go okay. Anyone got a clue?

**Comment by p0psicles** [comment]
There is a sonology DSM 7 Medusa package from @BenjV that's advised over docker.
